Makefile

```diff
@@ -83,6 +83,7 @@ $(DISTDIR)/gdbinit.py: language/generate.py language/lexer.py language/parser.py
 	$(VERBOSE)cp -a $(PWD)/contrib/gnatcoll-bindings $(DISTDIR)/
```

Comment:
We should consider using [build](https://pypa-build.readthedocs.io/en/latest/) instead of executing `setup.py` directly (which is considered as the legacy approach).

```diff
diff --git a/Makefile b/Makefile
index 38e8315..451fbc9 100644
--- a/Makefile
+++ b/Makefile
@@ -70,7 +70,7 @@ dist: $(BUILDDIR)/RecordFlux-parser-$(VERSION).tar.gz
        @echo "============================================================================================================"
 
 $(BUILDDIR)/RecordFlux-parser-$(VERSION).tar.gz: $(DISTDIR)/gdbinit.py disttools/setup.py
-       $(VERBOSE)(cd $(DISTDIR) && python3 setup.py sdist --formats=gztar --quiet --dist-dir=$(BUILDDIR))
+       $(VERBOSE)(cd $(DISTDIR) && python3 -m build --sdist --outdir=$(BUILDDIR))
        $(VERBOSE)rm -rf $(DISTDIR)
        $(VERBOSE)ls -l $@
```

Reply:
It's hard to keep track of what's legacy and what's not yet legacy ;) Changed in 4189520.